### PR TITLE
fix:(DB/world) Re-arrange location of Webwinder Path's signpost gps so they sit on the sign

### PR DESCRIPTION
**Pull request by damnablebear**
Fixing the webwinder path signpost gps locations so they actually sit on the post rather than floating

## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11992

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. .tele webwinder
2. Walk a bit south and check the signpost, make sure the three pieces nailed on to the sign aren't floating in space and appear attached to the post.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by damnablebear** [comment]
Preview of the fix:

image

**Comment by Kitzunu** [comment]
This should ideally be sniffed.

Also please see https://www.azerothcore.org/wiki/cpp-code-standards

File: data/sql/updates/pending_db_world/webwinder-signpost-gps.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE `gameobject` SET `position_x` = '634.220', `position_y` =  '359.989' WHERE (`guid` = '47468' and id = '19569');
+UPDATE `gameobject` SET `position_x` = '634.065', `position_y` =  '360.295' WHERE (`guid` = '47480' and id = '19570');
+UPDATE `gameobject` SET `position_x` = '633.923', `position_y` =  '359.858', `position_z` = '50.713' WHERE (`guid` = '47481' and id = '19571');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/webwinder-signpost-gps.sql:3]:
```suggestion
UPDATE `gameobject` SET `position_x` = 633.923, `position_y` =  359.858, `position_z` = 50.713 WHERE `guid` = 47481 AND `id` = 19571;
```
and add whiteline at end. also same formatting for the others brackets don't really matter that much afaik but I left them out

**Comment by elthehablo** [comment]
Other lines too please :)

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
Going to merge it as the sign doesnt show in parses

**Pull request merged**